src/components/JournalAboutPreview/index.jsx

```diff
@@ -14,13 +14,13 @@ class JournalAboutPreview extends React.Component {
         journal={{
           finishedFetching: this.props.finishedFetching,
           title: this.props.previewPage.title,
-          heroImageUrl: this.props.previewPage.hero_image_url,
```

Comment:
Curious about all these snake_case to camelCase properties. `this.props.previewPage` points to `state.preview.page` which is the stored response from `${JournalsApiService.apiUrl}/preview/${previewId}`. How to this properties correctly map without changing the API response to camelCase?

Reply:
@tuchfarber Zia had also raised this concern. So, I tried to run the preview page it was working fine. But now I dig a bit deeper and it seems like it was not working correctly instead it was getting the by default values. Now, I have taken care of that in `JournalAboutPreviewContainer/index.jsx`. Let me know if there is something missing.

Reply:
Discussed this one with the front end team because we've kind of let snake_case leak throughout the code on this project. If we're going to change the attribute names, we should do it in [the reducer](https://github.com/edx/journals-frontend/blob/master/src/data/reducers/preview.js). You can do a similar thing as you did in the container class. It'll just be better to have all API response manipulation in the reducer. This will change any upstream preview components that use that data though, so I'm fine with either updating them in this PR, or pulling out the changing of cases into a second PR. Let me know if you have any questions.

Reply:
@tuchfarber Doing this in reducer that was my first impression. But there were two reasons for not doing this in reducer.

1. Both `JournalPreview` and `JournalAboutPreview` using the same reducer. I don't want to seperate them for manupulation the data.  
2. We are also doing the same kind of thing in [JournalPreviewContainer](https://github.com/edx/journals-frontend/blob/27442e60175658e6b4afc1c3ffa2fbfb3b519f96/src/containers/JournalPreviewContainer/index.jsx#L9). Our code will be consistent.

But if you still think, we shouldn't do this work in container let me know then I am gonna change the PR according to original code.

Reply:
@tuchfarber I have added the new reducer. Let me know your thoughts about that.